### PR TITLE
Adaption for document query impl

### PR DESCRIPTION
**Pull request by fmatthies**

### CODE REVIEW

**Comment by fmatthies** [comment]
Conflicts are the versions (I made an increase 0.9.1 -> 0.9.2)

**Review by ChristophB — APPROVED** [review]

File: schemas/top-api.yml [hunk 1]
```diff
@@ -193,6 +195,24 @@ paths:
           $ref: '#/components/responses/UnexpectedError'
       tags:
         - document
+  '/document/query/{organisationId}/{repositoryId}/{queryId}':
```
Comment (ChristophB) [inline schemas/top-api.yml:198]: Is this path still used? I don't see the difference to `getQueryResultIds`.

Comment (ChristophB) [reply schemas/top-api.yml:198]: If you make any changes to address this comment please note that I already rebased your branch to have a consistent history.

Comment (fmatthies) [reply schemas/top-api.yml:198]: it's a little messy, but they return two different results (one DocumentPages, the other a list of ids). I renamed the former to be less confusing. Maybe this needs to be consolidated in the future?

Comment (fmatthies) [reply schemas/top-api.yml:198]: > If you make any changes to address this comment please note that I already rebased your branch to have a consistent history.

Something is not right with my rebase workflow. I rebased my branch as well before the commit (at least I thought so). And it always seems like a real pain.

**Review by ChristophB — APPROVED** [review]

**Pull request merged**